components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -143,7 +143,7 @@ - (void)testSnackbarSetAccessibiltyViewIsModalForActionSnacbars {
   XCTAssertTrue(self.manager.internalManager.overlayView.accessibilityViewIsModal);
 }
 
-- (void)testSnackbarAccessibiltyViewIsModalShouldBeNoWithNoActions {
+- (void)testSnackbarAccessibiltyViewIsModalShouldBeYesWithNoActions {
```

Comment:
@yarneo said,
> in the test they are explicitly setting shouldEnableAccessibilityViewIsModal = YES. I think the name of the test is confusing and makes it sound like the default is YES.

Agreed. Please consider renaming to something like "testAccessibilityViewIsModalAppliedToOverlayView".  If there's already a test like that, we can probably delete this one.